versions.gradle

```diff
@@ -9,7 +9,7 @@ ext {
     versionCode = 58
     versionName = '3.2.3'
     minSdkVersion = 16
-    targetSdkVersion = 29
-    compileSdkVersion = 29
+    targetSdkVersion = 30
```

Comment:
Did we test other parts of the SDK?

Reply:
Reverted. I've created a [work item](https://msmobilecenter.visualstudio.com/Mobile-Center/_boards/board/t/IvanM-Team/Backlog%20Items/?workitem=81505) to check Android 11 changes, but we haven't done the thorough testing yet.